docs/manuals/exporters/image-registry.md

```diff
@@ -19,7 +19,7 @@ $ docker buildx build --output type=registry[,parameters] .
 The following table describes the available parameters that you can pass to
 `--output` for `type=image`:
 
-| Parameter              | Value                                  | Default | Description                                                                                                                                                                                                                         |
+| Parameter              | Type                                   | Default | Description                                                                                                                                                                                                                         |
 | ---------------------- | -------------------------------------- | ------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
 | `name`                 | String                                 |         | Specify image name(s)                                                                                                                                                                                                               |
 | `push`                 | `true`,`false`                         | `false` | Push after creating the image.                                                                                                                                                                                                      |
```

Comment:
Maybe Type `Bool` for `push` would make more sense?

Reply:
WDYT @dvdksn? Our actions seem to use `Bool`, these docs use `true,false` - any preference? I think `true,false` is more explicit in the options that are available, while `Bool` is shorter and more succinct.

Reply:
I was secretly hoping y'all had a strong preference for either because I don't 🙈 But IMO let's stick with `true`,`false` for now since it is more explicit.